src/hitchhiker/tree/messaging.cljc

```diff
@@ -0,0 +1,252 @@
+(ns hitchhiker.tree.messaging
```

Comment:
Could you do the same as for `core.clj` and make a copy in `messaging.cljc` in the meantime?

Another thought that occurs to me is that I could certainly accept a PR that changes `messaging.clj` to `messaging.cljc`, and then this PR could be rebased onto that. If you're not sure how to do that, I'd be happy to teach you.